### PR TITLE
migrated two enums out

### PR DESCRIPTION
**Pull request by crutkas**
## PR checklist

Quick summary of changes

- Which issue does this PR relate to?
#890 
- Anything that requires particular review or attention?
nope
- Do all automated tests pass?
yes

### CODE REVIEW

File: code/src/UI/Controls/CodeViewer.cs [hunk 1]
```diff
@@ -11,6 +11,7 @@
 
 using Microsoft.Templates.UI.Services;
 using Microsoft.Templates.UI.ViewModels.NewItem;
+using Microsoft.Templates.UI.ViewModels.Common;
```
Comment (dotMorten) [inline code/src/UI/Controls/CodeViewer.cs]: Why add usings when no new code changes seems to require them?

File: code/src/UI/ViewModels/Common/FileExtension.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+{
+    public enum FileExtension
+    {
+        Default,
+        CSharp,
+        Resw,
+        Xaml,
+        Xml,
+        Csproj,
+        Appxmanifest,
```
Comment (dotMorten) [inline code/src/UI/ViewModels/Common/FileExtension.cs]: Shouldn't this be `AppXManifest`?

File: code/src/UI/ViewModels/Common/FileStatus.cs [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Templates.UI.ViewModels.Common
+{
+    public enum FileStatus
+    {
+        NewFile,
+        ModifiedFile,
+        ConflictingFile,
+        WarningFile,
```
Comment (dotMorten) [inline code/src/UI/ViewModels/Common/FileStatus.cs]: Remove the 'File' portion from all 4. That's implied.

File: code/src/UI/ViewModels/Common/FileExtension.cs (hunk 2 above)
Comment (crutkas) [inline code/src/UI/ViewModels/Common/FileExtension.cs]: Fair.  I just migrated. I will adjust

File: code/src/UI/Controls/CodeViewer.cs (hunk 1 above)
Comment (crutkas) [inline code/src/UI/Controls/CodeViewer.cs]: I strongly believe we will need to make this a common shared item after we do some refactoring. It was tightly tied to a VM to right click add

Comment (crutkas) [inline code/src/UI/Controls/CodeViewer.cs]: Actually feel this should be in core, not the UI project.

File: code/src/UI/ViewModels/Common/FileStatus.cs (hunk 3 above)
Comment (crutkas) [inline code/src/UI/ViewModels/Common/FileStatus.cs]: Fair.  Did migration.  Can refactor

**Comment by crutkas** [comment]
@dotMorten adjusted

**Pull request merged**